From 2aa4d449992352f5c130ec1b5e4aad39536e0095 Mon Sep 17 00:00:00 2001
From: Ecco park <eccopark@broadcom.com>
Date: Wed, 3 Sep 2014 14:06:05 -0700
Subject: [PATCH 68/77] net: wireless: bcmdhd set random mac for SOFTAP

17373032 SoftAp has constant MAC that is not changed between reboots

Change-Id: I3cd0283175e47391e5def68cf1c6cc375b777163
Signed-off-by: Ecco park <eccopark@broadcom.com>
Signed-off-by: Dmitry Shmidt <dimitrysh@google.com>
---
 drivers/net/wireless/bcmdhd/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/wireless/bcmdhd/Makefile b/drivers/net/wireless/bcmdhd/Makefile
index 1d91d14b5dc0..a56b569ba21c 100644
--- a/drivers/net/wireless/bcmdhd/Makefile
+++ b/drivers/net/wireless/bcmdhd/Makefile
@@ -98,6 +98,7 @@ DHDCFLAGS += -DWL_ENABLE_P2P_IF
 
 DHDCFLAGS += -DWL_CFG80211_ACL
 DHDCFLAGS += -DDISABLE_11H_SOFTAP
+DHDCFLAGS += -DSET_RANDOM_MAC_SOFTAP
 
 
 ##########################
-- 
2.0.3

